jackal_control/launch/control.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
-  <rosparam command="load" file="$(find jackal_control)/config/control.yaml" />
+  <rosparam file="$(find jackal_control)/config/control.yaml" command="load"/>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="jackal_joint_publisher jackal_velocity_controller --shutdown-timeout 1" />
-  
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" /> 
+
+  <include file="$(find jackal_control)/launch/robot_localization.launch"/>
```

Comment:
Well, I basically replaced a `node` with a `include`, so I kept it inside `control.launch`. :)

The reason for the new `launch` is to packed it with the params so it can be used from other packages.
Not a very a strong reason, I admit, but centralizes a single `robot_localization.launch` here.

I think the reason to keep the `ekf_localization` (that I renamed to `t1_ekf_localization` to avoid collision if a second ekf is used) inside `control.launch` is to have the `odom` frame published on TF with the minimal deployment.
Anyway, if you prefer to have `control.launch` w/o the nested `robot_localization.launch`, I can include it on the two other launch files you're mentioning. At the end it'll be the same.

BTW, although I don't use it too much, I think [rqt_launch](http://wiki.ros.org/rqt_launch) helps with nested launch files.